index.js

```diff
@@ -60,6 +60,56 @@ const getWslDrivesMountPoint = (() => {
 	};
 })();
 
+/**
+ * Get the default browser name in Windows from WSL.
+ * @returns {Promise<string>} Browser name
+ */
+async function getWindowsDefaultBrowserFromWsl() {
+	const mountPoint = await getWslDrivesMountPoint();
+	const powershellPath = `${mountPoint}c/Windows/System32/WindowsPowerShell/v1.0/powershell.exe`;
+
+	const rawCommand
+		= `$prog = Get-ItemProperty -Path 'HKCU:\\Software\\Microsoft\\Windows\\Shell\\Associations\\UrlAssociations\\http\\UserChoice' | Select-Object -ExpandProperty ProgId
+	Write-Output $prog
+	`.trim();
+	const encodedCommand = Buffer.from(rawCommand, 'utf16le').toString('base64');
+
+	return new Promise((resolve, reject) => {
+		execFile(
```

Comment:
Use the promise version: https://nodejs.org/api/child_process.html#:~:text=If%20this%20method%20is%20invoked%20as%20its%20util.promisify()ed%20ver,-sion